### ISSUE TITLE
Two 'selects' can be open by clicking on an arrow

### ISSUE THREAD

**Issue by an4ger** [JedWatson/react-select#2182]
I have 2 selects on one page. Click on first select arrow and than click on second select arrow. Both selects stay opened.

![react-select-2opened](https://user-images.githubusercontent.com/5645926/33377756-3fd93a00-d51b-11e7-82a1-406092dc4372.png)

Problem is located in: [https://github.com/JedWatson/react-select/commit/f6920deefd935d366cddff14cb45b7f6f950c08b](https://github.com/JedWatson/react-select/commit/f6920deefd935d366cddff14cb45b7f6f950c08b)

Return statement was removed and event not bubble to main handleMouseDown

```
 		if (!this.state.isOpen) {
-			return;
+			this.setState({
+				isOpen: true,
+			});
 		}
 		// prevent default event handlers
 		event.stopPropagation();
 		event.preventDefault();
```

**Comment by kurtinatlanta** [comment]
I submitted PR #2213 to fix this.

**Comment by JedWatson** [comment]
Fixed by #2213 with thanks to @kurtinatlanta

**Comment by vladlucapitech** [comment]
Thank's, when to expect a new release that contains this fix?
